js/src/forum/components/DiscussionPage.js

```diff
@@ -41,7 +42,7 @@ export default class DiscussionPage extends Page {
       app.pane.enable();
       app.pane.hide();
 
-      if (app.previous instanceof DiscussionPage) {
+      if (app.previous && app.previous.subclassOf(DiscussionPage)) {
```

Comment:
```suggestion
      if (subclassOf(app.previous, DiscussionPage)) {
```

Reply:
Oh wait, you really did add that method to the `PageState` class. Never mind my suggestion then.

Would it be possible to ensure that `app.current` and `app.previous` are *always* set (at least to some kind of [null object](https://en.wikipedia.org/wiki/Null_object_pattern)) so that we can omit these long conditionals?

Reply:
That said, the `subclassOf` import is no longer needed. Here and elsewhere...